R/pkg/tests/fulltests/test_sparkSQL.R

```diff
@@ -3657,8 +3657,7 @@ test_that("Call DataFrameWriter.load() API in Java without path and check argume
   # It makes sure that we can omit path argument in read.df API and then it calls
   # DataFrameWriter.load() without path.
   expect_error(read.df(source = "json"),
-               paste("Error in load : analysis error - Unable to infer schema for JSON.",
-                     "It must be specified manually"))
+               "Error in load : analysis error - Unable to infer schema for")
```

Comment:
hmm, it doesn't say "JSON" now?

Reply:
The current error message is changed to `Unable to infer schema for $tableName`, while the `tableName`is `shortName + path`. I can create another PR to fix that.